packages/components/molecules/f-rating/README.md

```diff
@@ -69,8 +69,9 @@ There may be props that allow you to customise its functionality.
 
 The props that can be defined are as follows (if any):
 
-| Prop  | Type  | Default | Description |
-| ----- | ----- | ------- | ----------- |
+| Prop          |   Type   | Required | Default | Description                       |
+| :---          |:--------:|:--------:|:-------:|:----------------------------------|
+| `starRating`  | `Number` |   Yes    |    -    | Sets the number of stars out of 5 |
```

Comment:
More out of interest than anything else. Are we migrating all platforms to use 5 stars using this component, or will there be a use-case for 6 stars in the interim?

Reply:
That's my understand. Anyone on 6 probably will keep their current implementation until they move across although i believe they may have no choice if the api is being worked on.